### ISSUE TITLE
Configure django-environ (do this after implementing `local_settings` from Heroku tutorial)

### ISSUE THREAD

**Issue by webknjaz** [kpi-web-guild/django-girls-blog-slsh1o#27]
### The math
Read about the usage of environment variables in PaaS and heroku/travis for deployment.

Ref: https://12factor.net/

### The task
* [ ] Implement single `settings.py` using [django-environ](https://django-environ.readthedocs.io/en/latest/)
  * [ ] Remove everything related to `local_settings.py`
* [ ] Make every option differing at development and production be read from environment variables
  * [ ] DEBUG
  * [ ] SECRET
  * [ ] db config
  * [ ] use `env.Path` for constructing fs pathes
  * [ ] etc.
* [ ] Enable reading env vars from dotenv (`.env`) file
  * [ ] .gitignore it
  * [ ] add `.env.example` file listing required env vars with dummy values for the other dev as a quickstart

**Comment by slsh1o** [comment]
@webknjaz I don't understend why do I need `local_settings.py` file if I can put everything in `.env` file? Or I missing out something?

Or by _Remove everything related to local_settings.py_ you mean that I should also remove `local_settings.py` file and switch to `django-environ` with `.env` file?

**Comment by webknjaz** [comment]
Yes, you should remove `local_settings.py` when migrating to `.env`. But the previous step is completing the Heroku task that adds it. This gives you a chance to compare the approaches.

**Comment by webknjaz** [comment]
I should've probably added an issue with https://tutorial-extensions.djangogirls.org/en/heroku/ first.
